web_src/less/_repository.less

```diff
@@ -1444,6 +1444,66 @@
     }
   }
 
+  .data-table {
+    width: 100%;
+
+    tr {
+      border-top: 0;
+    }
+
+    td,
+    th {
+      padding: 5px !important;
+      overflow: hidden;
+      font-size: 12px;
+      line-height: 1;
```

Comment:
I would use something a little higher, like 1.2. Gets a little squished with some fonts.

![image](https://user-images.githubusercontent.com/12700993/112229102-f5166280-8bf7-11eb-9f50-a31ade1a7431.png)

Reply:
I have removed the line-height. Looks better without it.